core/controllers/topic_editor.py

```diff
@@ -163,14 +165,30 @@ def get(self, topic_id):
             raise self.PageNotFoundException(
                 Exception('The topic with the given id doesn\'t exist.'))
 
-        skill_ids = topic.get_all_skill_ids()
-
-        skill_id_to_description_dict = (
-            skill_services.get_skill_descriptions_by_ids(topic_id, skill_ids))
+        skill_id_to_description_dict, deleted_skill_ids = (
+            skill_services.get_descriptions_of_skills(
+                topic.get_all_skill_ids()))
+
+        skill_id_to_rubrics_dict, deleted_skill_ids = (
+            skill_services.get_rubrics_of_skills(topic.get_all_skill_ids())
+        )
+
+        if deleted_skill_ids:
+            deleted_skills_string = ', '.join(deleted_skill_ids)
+            logging.error(
+                'The deleted skills: %s are still present in topic with id %s'
+                % (deleted_skills_string, topic_id)
+            )
+            if feconf.CAN_SEND_EMAILS:
```

Comment:
I am worried that this may cause a lot of spam emails to the admin? For now it is fine but in the future we should consider batching this. Or we can use a google group instead to report this that people can subscribe to.

Reply:
From just using the editors, this should never occur, as skills can only be deleted once removed from the topic. So, the mails would be sent, if at all, very rarely.
I'll create an issue to maybe batch this, is that fine?

Reply:
That's fine!